src/Hl7.Fhir.Specification/Specification/Terminology/ITerminologyService.cs

```diff
@@ -7,11 +7,75 @@
  */
 
 using Hl7.Fhir.Model;
+using System;
 
 namespace Hl7.Fhir.Specification.Terminology
 {
     public interface ITerminologyService
```

Comment:
I know I am a bit late in realizing this, but shouldn't all these new methods be async?

Reply:
Sigh, you are right. But a lot of work. I'll try to do this today

Reply:
sorry 😢